README.md

```diff
@@ -63,6 +63,52 @@ roll_up_capture(returns, window=60)
 
 Please [open an issue](https://github.com/quantopian/empyrical/issues/new) for support.
 
+### Deprecated: Data Reading via `pandas-datareader`
+
+As of early 2018, Yahoo Finance has suffered major API breaks with no stable
+replacement, and the Google Finance API has not been stable since late 2017
+[(source)](https://github.com/pydata/pandas-datareader/blob/da18fbd7621d473828d7fa81dfa5e0f9516b6793/README.rst).
+In recent months it has become a greater and greater strain on the `empyrical`
+development team to maintain support for fetching data through
+`pandas-datareader` and other third-party libraries, as these APIs are known to
+be unstable.
+
+As a result, all `empyrical` support for data reading functionality has been
+deprecated and will be removed in a future version.
+
+Users should beware that the following functions are now deprecated:
+
+- `empyrical.utils.cache_dir`
+- `empyrical.utils.data_path`
+- `empyrical.utils.ensure_directory`
+- `empyrical.utils._1_bday_ago`
+- `empyrical.utils.get_fama_french`
+- `empyrical.utils.load_portfolio_risk_factors`
+- `empyrical.utils.default_returns_func`
+- `empyrical.utils.get_symbol_returns_from_yahoo`
+
+Users should expect regular failures from the following functions, pending
+patches to the Yahoo or Google Finance API:
+
+- `empyrical.utils.default_returns_func`
+- `empyrical.utils.get_symbol_returns_from_yahoo`
+
+For alternative data sources, we suggest the following:
+
+1. Migrate your research workflow to the Quantopian Research environment,
```

Comment:
Yea I agree...I've been thinking about some similar questions with qgrid since we're going to direct people to research from there.  It feels OK to me in the qgrid case because we're going to be using research to host a demo of qgrid.  In this case it feels a bit weird because we're not directing them to anything empyrical-specific within research, we're just saying here's another place you can get data.

Reply:
I don't know if making the statement less imperative would help it make it less of an advertisement.
i.e. "we suggest" -> "these options are available".

I lean on the side of removing, so that we don't have to worry about semantics.

Reply:
Okay, removed all suggestions.